app/models/alchemy/language.rb

```diff
@@ -38,7 +38,7 @@ class Language < BaseRecord
 
     validates :language_code,
       presence: true,
-      uniqueness: { scope: [:site_id, :country_code] },
+      uniqueness: { scope: [:site_id, :country_code], case_sensitive: false },
```

Comment:
now my comment is outdated. the setting is now on the right place, but still it needs to be true